src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -44,6 +44,9 @@ const propTypes = {
     /** Does this fragment belong to a reportAction that has not yet loaded? */
     loading: PropTypes.bool,
 
+    /** The reportAction's source */
+    source: PropTypes.string,
```

Comment:
Should we make this more specific like `PropTypes.oneOf(['email', ...])`?

What other sources could there be?

Reply:
'Chronos': https://github.com/Expensify/Web-Expensify/blob/2f46bbb7f189948b8aac833fed097a9f690446ca/lib/Chronos/Chronos.php#L60, and a few other places
'email': https://github.com/Expensify/Web-Expensify/blob/2f46bbb7f189948b8aac833fed097a9f690446ca/lib/Mailgun/API.php#L1330
'ios', 'android', 'web', 'email': https://github.com/Expensify/Web-Expensify/blob/2f46bbb7f189948b8aac833fed097a9f690446ca/lib/ChatBot/Message.php#L98-L101

Not sure if there could be others... 'Chronos' is not in a constant.

If you use this: `PropTypes.oneOf(['email', ...])`, does it crash if a source doesn't match that list? or will it just warn?

Reply:
<img width="573" alt="image" src="https://user-images.githubusercontent.com/87341702/176305326-ddd5b2e9-2bd9-4ba5-8e77-0da62565531f.png">
Just a warning, no crash, so I think we can safely do that

Reply:
Seems like a good idea to me, but NAB

Reply:
Changed to: `PropTypes.oneOf(['Chronos', 'email', 'ios', 'android', 'web', 'email', '']),`